ui/src/app/utils/aou_external_links.tsx

```diff
@@ -2,5 +2,9 @@ export const CROMWELL_INFORMATION_LINK =
   'https://support.researchallofus.org/hc/en-us/articles/14428263737620';
 export const CROMWELL_INTRO_LINK =
   'https://cromwell.readthedocs.io/en/stable/tutorials/FiveMinuteIntro/';
+export const RSTUDIO_HOW_TO_RUN_LINK =
+  'https://support.researchallofus.org/hc/en-us/articles/22078658566804';
```

Comment:
I'm not sure that we want to hard-code helpdesk links like this.  We have a Zendesk module we can use instead, for future flexibility.

For example, we can refer to `supportUrls.createBillingAccount` which points to the correct "create billing account" article for each environment.  It does this by specifying `createBillingAccount: article('360039539411')` in `zendesk.ts` 

Here we can do something similar, `article('22078658566804')`

Reply:
Good call! I updated both the RStudio and Cromwell links to work this way.